book/src/pong/03_sprites.md

```diff
@@ -0,0 +1,175 @@
+# Sprites
+
+In this section, we'll put the sprites needed for our pong game onto the screen.
+We'll cover what sprites are in the Game Boy Advance, and how to get them to show up on screen.
+We'll briefly cover vblank and by the end of this section, you'll have a ball bouncing around the screen!
+
+# Why do we need sprites in the first place?
+
+The Game Boy Advance has a 240x160px screen, with 15-bit RGB colour support.
+In order to manually set the colour for each pixel in the screen, you would need to update a total of 38,400 pixels per frame, or 2,304,000 pixels per second at 60 fps.
+With a 16MHz processor, that means you would need to be able to calculate 1 pixel every 8 clock cycles, which is pretty much impossible.
+You could get clever with how you update these pixels, but there is a much easier way which almost every game for the Game Boy Advance uses.
```

Comment:
Use of singular "there is a much easier way", but then give two ways.

Reply:
Updated to `You could get clever with how you update these pixels, but using the tools provided by the Game Boy Advance to put pixels on the screen, you'll have a much easier time.`